spec/controllers/exhibits_controller_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.describe ExhibitsController, vcr: { cassette_name: "all_collections" } do
+RSpec.describe ExhibitsController, vcr: { cassette_name: "collections_manifest" } do
```

Comment:
Now that it's actually ingesting the members, it needed to fetch additional pages.  But now that I look at the recorded URLs, I think the new cassette can just replace the old one instead of being separate.  So I'll just move collections_manifest to all_collections.